### PR TITLE
Indent fixes

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Coses #2844.

### CODE REVIEW

File: tests/Fixtures/Integration/misc/braces,phpdoc_indent.test [hunk 1]
```diff
@@ -0,0 +1,14 @@
+--TEST--
+Integration of fixers: braces,phpdoc_indent.
```
Comment (keradus) [inline tests/Fixtures/Integration/misc/braces,phpdoc_indent.test]: instead of integration test write proper utest. `braces` has nothing to do here

Comment (GrahamCampbell) [inline tests/Fixtures/Integration/misc/braces,phpdoc_indent.test]: it doesn't fail when we just run one fixer, only when we run them both, and its the braces fixer that crashes!

Comment (keradus) [inline tests/Fixtures/Integration/misc/braces,phpdoc_indent.test]: it fails, but not crash, when run over one fixer.
any fixer that would run after phpdoc_indent could crash, as phpdoc_intent leaves collection in invalid state. as told in other pr and here, this has nothing to do with that particular rules combination.

**Comment by keradus** [comment]
replaced by #2848

File: tests/Fixtures/Integration/misc/braces,phpdoc_indent.test (hunk 1 above)
Comment (GrahamCampbell) [inline tests/Fixtures/Integration/misc/braces,phpdoc_indent.test]: > it fails, but not crash, when run over one fixer.

Not when I run the tool itself. It finishes with OK for both fixers separately.

Comment (keradus) [inline tests/Fixtures/Integration/misc/braces,phpdoc_indent.test]: yet one of them leave the collection in invalid state. that's why i said fail, not crash

**Pull request closed without merging**